Meadowlark-js/tests/config/global-setup.js

```diff
@@ -1 +1 @@
-jest.setTimeout(40000)
+jest.setTimeout(100000)
```

Comment:
Why was it necessary to increase the default timeout?

Reply:
Undoing this change.